interface/globals.php

```diff
@@ -465,4 +465,23 @@ function strterm($string,$length) {
   extract($_GET,EXTR_SKIP);
   extract($_POST,EXTR_SKIP);
 }
+//Create page validation array -- all the pages that have to fire validatejs and their rules
+function collectValidationPageRules($option_id,$title){
+    if (!empty($_SESSION['authUserID'])) {
+
+
+        $pages = sqlStatement("SELECT * " .
+            "FROM `list_options` WHERE list_id='PageValidation' AND option_id='$option_id' AND title='$title'");
+
+          $row = sqlFetchArray($pages);
+          if($row) {
+              return array('page_name' => $row['option_id'] . ".php", 'rules' => $row['notes']);
+          }
+          else{
+              return null;
+          }
+
```

Comment:
The problem is that the paths have a real chance of changing as we modernize the code. The validator_id, however will not change. Also note you can assign 1 validator_id to multiple pages (if similar forms etc.), by placing multiple paths in the title entry. The title entry is really just meant for folks knowing what scripts the validator_id (ie. option_id) is being used at a quick glance in administration->lists->PageValidation.